docs/concepts.md

```diff
@@ -141,3 +177,15 @@ Another typical Monitoring Set would be that which nominates the complete
 footprint for the Transmission Provider. A natural choice for the
 `monitoring-set` identifier is the NERC id of the entity that defines the
 `monitoring-set`, if applicable.
+
+## Seasons
```

Comment:
Can we pull this out for now? I'd like to have a design discussion about Seasons in interop. Let's add this concept as part of the Seasonal milestone.

Reply:
I think you're right.  I was trying to be complete, but I realize I'm kind of expanding the scope too much here.